lib/govuk_publishing_components/presenters/checkboxes_helper.rb

```diff
@@ -25,6 +25,10 @@ def initialize(options)
         @visually_hide_heading = options[:visually_hide_heading]
       end
 
+      def should_have_fieldset
+        true if @items.length > 1 || @heading
```

Comment:
In relation to my other comment, perhaps this should be:

`true if @items.length > 1`

Reply:
I don't think that would work as we don't want items with, for example, 5 checkboxes but no heading to fall through to the `else` condition?

Reply:
Yeah, this is really complex. The logic is:

- if one checkbox, don't render a fieldset and a legend
- if one checkbox and a heading is supplied, render a fieldset and a legend (using the heading)
- if more than one checkbox, and no heading, fail

Tried a few options for this but so far nothing works. I've updated the helper method with a little comment to explain. If I think of a better way of doing this in future I'll definitely change it but for now I think it's okay.